docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md

```diff
@@ -1 +1,150 @@
-# Working with Salesforce client
+Template for Working with Salesforce client
+
+# Working with Salesforce client 
+
+This template refers to the tutorial described in [Working with Salesforce client](https://ei.docs.wso2.com/en/latest/ballerina-integrator/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/1/). Please refer it for more details on what you are going to build here and the prerequisites for building the module .
+
+#### Executing the Template
+
+- Run the following command to pull the ```working_with_salesforce_client``` template from the Ballerina Central.
+
+```
+$ ballerina pull wso2/working_with_salesforce_client
+```
+
+- Create a new project.
+
+```bash
+$ ballerina new working-with-salesforce-client
+```
+
+- Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.
+
+```bash
+$ ballerina add -t wso2/working_with_salesforce_client working_with_salesforce_client
+```
+
+This automatically creates a working_with_salesforce_client for you inside the `src` directory of your project .  
+
+### Testing
+
+First let’s build the module. Navigate to the project root directory and execute the following command.
+
+```bash
+$ ballerina build working_with_salesforce_client 
+```
+
+This creates the executables. Now run the `working_with_salesforce_client.jar` file created in the above step.
+
+```bash
+$ java -jar target/bin/working_with_salesforce_client.jar
+```
+
+You will see service the following log.
+
+```log
+[ballerina/http] started HTTP/WS listener 0.0.0.0:9090
+```
+
+#### Test createAccount resource
```

Comment:
Remove everything below this point. They should get this information from the docs.

Reply:
Fixed